napari_animation/_qt/animation_widget.py

```diff
@@ -122,6 +126,30 @@ def _on_keyframes_changed(self, event=None):
         self.keyframesListWidget.setEnabled(has_frames)
         self.frameWidget.setEnabled(has_frames)
 
+    def _on_active_keyframe_changed(self, event=None):
+        n_frames = len(self.animation._frames)
+        active_keyframe = event.value
+
+        if active_keyframe and n_frames:
+            self.animationSlider.blockSignals(True)
+            kf1_list = [
+                self.animation._frames._frame_index[n][0]
+                for n in range(n_frames)
+            ]
+            frame_index = kf1_list.index(active_keyframe)
+            self.animationSlider.setValue(frame_index)
+            self.animationSlider.blockSignals(False)
+
+        self.keyframesListControlWidget.deleteButton.setEnabled(
+            bool(active_keyframe)
+        )
+
+    def _slider_moved(self, event=None):
```

Comment:
could you rename this `_on_slider_moved` to be consistent with the other methods?